### ISSUE TITLE
Cannot install omnipose with GUI

### ISSUE THREAD

**Issue by bpavie** [kevinjohncutler/omnipose#53]
Hi 

I tried to install omnipose and run it with the GUI but it failed.

**Environment:**
- Windows 11
- mamba 1.4.2
- conda 23.1.0

First I created the environment
```
mamba create -n omnipose pytorch cudatoolkit=11.3 -c pytorch
mamba activate omnipose
mamba install -c https://conda.anaconda.org/conda-forge mahotas
```
Note: I installed mamba via conda since via pip I add some issues

Then I installed omnipose
```
pip install omnipose
Collecting omnipose
  Using cached omnipose-0.4.4-py3-none-any.whl (61 kB)
Requirement already satisfied: numpy>=1.22.4 in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from omnipose) (1.25.1)
Collecting scipy (from omnipose)
  Obtaining dependency information for scipy from https://files.pythonhosted.org/packages/04/b8/947f40706ee2e316fd1a191688f690c4c2b351c2d043fe9deb9b7940e36e/scipy-1.11.1-cp311-cp311-win_amd64.whl.metadata
  Using cached scipy-1.11.1-cp311-cp311-win_amd64.whl.metadata (59 kB)
Collecting numba (from omnipose)
  Obtaining dependency information for numba from https://files.pythonhosted.org/packages/40/db/3a08f2ec91e0d6b0c68d94a42e80af005a0378c83c63cbbc9ee63143f2ed/numba-0.57.1-cp311-cp311-win_amd64.whl.metadata
  Using cached numba-0.57.1-cp311-cp311-win_amd64.whl.metadata (2.8 kB)
Collecting edt (from omnipose)
  Using cached edt-2.3.1-cp311-cp311-win_amd64.whl (215 kB)
Collecting scikit-image (from omnipose)
  Obtaining dependency information for scikit-image from https://files.pythonhosted.org/packages/08/c0/8085c5fd2f7f7514a0c5031b666171d5828ac5b3c9cf5d0ecd19688d5407/scikit_image-0.21.0-cp311-cp311-win_amd64.whl.metadata
  Using cached scikit_image-0.21.0-cp311-cp311-win_amd64.whl.metadata (14 kB)
Collecting ncolor>=1.2.1 (from omnipose)
  Using cached ncolor-1.2.1-py3-none-any.whl (7.5 kB)
Collecting scikit-learn (from omnipose)
  Obtaining dependency information for scikit-learn from https://files.pythonhosted.org/packages/77/85/bff3a1e818ec6aa3dd466ff4f4b0a727db9fdb41f2e849747ad902ddbe95/scikit_learn-1.3.0-cp311-cp311-win_amd64.whl.metadata
  Using cached scikit_learn-1.3.0-cp311-cp311-win_amd64.whl.metadata (11 kB)
Requirement already satisfied: torch>=1.10 in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from omnipose) (2.0.1)
Requirement already satisfied: mahotas>=1.4.13 in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from omnipose) (1.4.13)
Collecting mgen (from omnipose)
  Using cached mgen-1.2.1-py3-none-any.whl (8.1 kB)
Collecting matplotlib (from omnipose)
  Obtaining dependency information for matplotlib from https://files.pythonhosted.org/packages/4d/9c/65830d4a56c47f5283eaa244dc1228c5da9c844a9f999ebcc2e69bf6cc65/matplotlib-3.7.2-cp311-cp311-win_amd64.whl.metadata
  Using cached matplotlib-3.7.2-cp311-cp311-win_amd64.whl.metadata (5.8 kB)
Collecting peakdetect (from omnipose)
  Using cached peakdetect-1.2-py2.py3-none-any.whl (9.7 kB)
Collecting igraph (from omnipose)
  Obtaining dependency information for igraph from https://files.pythonhosted.org/packages/e4/2c/aea4c9ff8d33c1ba45d3964d758019097c363363c1a67dbada9d9f98b3d9/igraph-0.10.6-cp39-abi3-win_amd64.whl.metadata
  Using cached igraph-0.10.6-cp39-abi3-win_amd64.whl.metadata (3.9 kB)
Collecting torchvf (from omnipose)
  Obtaining dependency information for torchvf from https://files.pythonhosted.org/packages/62/c9/82c150cf7151b06c0a4a30eabe1b3ed509beabe46ab031ece182e7058edf/torchvf-0.1.3-py3-none-any.whl.metadata
  Using cached torchvf-0.1.3-py3-none-any.whl.metadata (8.1 kB)
Collecting cellpose-omni[omni]>=0.9.1 (from omnipose)
  Using cached cellpose_omni-0.9.1-py3-none-any.whl (200 kB)
Collecting natsort (from cellpose-omni[omni]>=0.9.1->omnipose)
  Obtaining dependency information for natsort from https://files.pythonhosted.org/packages/ef/82/7a9d0550484a62c6da82858ee9419f3dd1ccc9aa1c26a1e43da3ecd20b0d/natsort-8.4.0-py3-none-any.whl.metadata
  Using cached natsort-8.4.0-py3-none-any.whl.metadata (21 kB)
Collecting tifffile (from cellpose-omni[omni]>=0.9.1->omnipose)
  Obtaining dependency information for tifffile from https://files.pythonhosted.org/packages/2d/e5/cc8a8ca43685006bb3ca56fab60707f3f74700844b18634db0b1e8b4b93f/tifffile-2023.7.18-py3-none-any.whl.metadata
  Using cached tifffile-2023.7.18-py3-none-any.whl.metadata (32 kB)
Collecting tqdm (from cellpose-omni[omni]>=0.9.1->omnipose)
  Using cached tqdm-4.65.0-py3-none-any.whl (77 kB)
Collecting opencv-python-headless (from cellpose-omni[omni]>=0.9.1->omnipose)
  Obtaining dependency information for opencv-python-headless from https://files.pythonhosted.org/packages/60/e7/a6dedca760db7a4686fd3b6053487ad4ad4d8f0591295dc8a9cf35269c42/opencv_python_headless-4.8.0.74-cp37-abi3-win_amd64.whl.metadata
  Using cached opencv_python_headless-4.8.0.74-cp37-abi3-win_amd64.whl.metadata (19 kB)
Collecting fastremap (from cellpose-omni[omni]>=0.9.1->omnipose)
  Obtaining dependency information for fastremap from https://files.pythonhosted.org/packages/53/4a/467e0eaf9e094d30560cea92670ca4983b92922b8063ad1086bcf8953cf8/fastremap-1.13.5-cp311-cp311-win_amd64.whl.metadata
  Using cached fastremap-1.13.5-cp311-cp311-win_amd64.whl.metadata (9.3 kB)
Collecting imagecodecs (from cellpose-omni[omni]>=0.9.1->omnipose)
  Obtaining dependency information for imagecodecs from https://files.pythonhosted.org/packages/0b/f9/7523da2ae1dd6d767e8f0455216bd9da37cf7cfa28dd0cdaab098bec1202/imagecodecs-2023.7.10-cp311-cp311-win_amd64.whl.metadata
  Using cached imagecodecs-2023.7.10-cp311-cp311-win_amd64.whl.metadata (20 kB)
Collecting torch-optimizer (from cellpose-omni[omni]>=0.9.1->omnipose)
  Using cached torch_optimizer-0.3.0-py3-none-any.whl (61 kB)
Requirement already satisfied: filelock in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from torch>=1.10->omnipose) (3.12.2)
Requirement already satisfied: typing-extensions in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from torch>=1.10->omnipose) (4.7.1)
Requirement already satisfied: sympy in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from torch>=1.10->omnipose) (1.12)
Requirement already satisfied: networkx in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from torch>=1.10->omnipose) (3.1)
Requirement already satisfied: jinja2 in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from torch>=1.10->omnipose) (3.1.2)
Collecting texttable>=1.6.2 (from igraph->omnipose)
  Using cached texttable-1.6.7-py2.py3-none-any.whl (10 kB)
Collecting contourpy>=1.0.1 (from matplotlib->omnipose)
  Obtaining dependency information for contourpy>=1.0.1 from https://files.pythonhosted.org/packages/16/09/989b982322439faa4bafffcd669e6f942b38fee897c2664c987bcd091dec/contourpy-1.1.0-cp311-cp311-win_amd64.whl.metadata
  Using cached contourpy-1.1.0-cp311-cp311-win_amd64.whl.metadata (5.7 kB)
Collecting cycler>=0.10 (from matplotlib->omnipose)
  Using cached cycler-0.11.0-py3-none-any.whl (6.4 kB)
Collecting fonttools>=4.22.0 (from matplotlib->omnipose)
  Obtaining dependency information for fonttools>=4.22.0 from https://files.pythonhosted.org/packages/bd/49/d921c3b1eca0c1a67093cbc2ce302b039df94dea9e246b43f95827f4f566/fonttools-4.41.1-cp311-cp311-win_amd64.whl.metadata
  Using cached fonttools-4.41.1-cp311-cp311-win_amd64.whl.metadata (153 kB)
Collecting kiwisolver>=1.0.1 (from matplotlib->omnipose)
  Using cached kiwisolver-1.4.4-cp311-cp311-win_amd64.whl (55 kB)
Collecting packaging>=20.0 (from matplotlib->omnipose)
  Using cached packaging-23.1-py3-none-any.whl (48 kB)
Collecting pillow>=6.2.0 (from matplotlib->omnipose)
  Obtaining dependency information for pillow>=6.2.0 from https://files.pythonhosted.org/packages/66/d4/054e491f0880bf0119ee79cdc03264e01d5732e06c454da8c69b83a7c8f2/Pillow-10.0.0-cp311-cp311-win_amd64.whl.metadata
  Using cached Pillow-10.0.0-cp311-cp311-win_amd64.whl.metadata (9.6 kB)
Collecting pyparsing<3.1,>=2.3.1 (from matplotlib->omnipose)
  Using cached pyparsing-3.0.9-py3-none-any.whl (98 kB)
Collecting python-dateutil>=2.7 (from matplotlib->omnipose)
  Using cached python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Collecting pbr (from mgen->omnipose)
  Using cached pbr-5.11.1-py2.py3-none-any.whl (112 kB)
Collecting llvmlite<0.41,>=0.40.0dev0 (from numba->omnipose)
  Obtaining dependency information for llvmlite<0.41,>=0.40.0dev0 from https://files.pythonhosted.org/packages/09/7a/3e0bec597ca9e3382ba87c139813ea0b05068d6d5358e05f4557a8c24d63/llvmlite-0.40.1-cp311-cp311-win_amd64.whl.metadata
  Using cached llvmlite-0.40.1-cp311-cp311-win_amd64.whl.metadata (4.8 kB)
Collecting numpy>=1.22.4 (from omnipose)
  Obtaining dependency information for numpy>=1.22.4 from https://files.pythonhosted.org/packages/d8/ec/ebef2f7d7c28503f958f0f8b992e7ce606fb74f9e891199329d5f5f87404/numpy-1.24.4-cp311-cp311-win_amd64.whl.metadata
  Using cached numpy-1.24.4-cp311-cp311-win_amd64.whl.metadata (5.6 kB)
Collecting imageio>=2.27 (from scikit-image->omnipose)
  Obtaining dependency information for imageio>=2.27 from https://files.pythonhosted.org/packages/c7/b0/7b6c35b8636ed773325cdb6f5ac3cd36afba63d99e20ed59c521cf5018b4/imageio-2.31.1-py3-none-any.whl.metadata
  Using cached imageio-2.31.1-py3-none-any.whl.metadata (4.7 kB)
Collecting PyWavelets>=1.1.1 (from scikit-image->omnipose)
  Using cached PyWavelets-1.4.1-cp311-cp311-win_amd64.whl (4.2 MB)
Collecting lazy_loader>=0.2 (from scikit-image->omnipose)
  Obtaining dependency information for lazy_loader>=0.2 from https://files.pythonhosted.org/packages/a1/c3/65b3814e155836acacf720e5be3b5757130346670ac454fee29d3eda1381/lazy_loader-0.3-py3-none-any.whl.metadata
  Using cached lazy_loader-0.3-py3-none-any.whl.metadata (4.3 kB)
Collecting joblib>=1.1.1 (from scikit-learn->omnipose)
  Obtaining dependency information for joblib>=1.1.1 from https://files.pythonhosted.org/packages/28/08/9dcdaa5aac4634e4c23af26d92121f7ce445c630efa0d3037881ae2407fb/joblib-1.3.1-py3-none-any.whl.metadata
  Using cached joblib-1.3.1-py3-none-any.whl.metadata (5.4 kB)
Collecting threadpoolctl>=2.0.0 (from scikit-learn->omnipose)
  Obtaining dependency information for threadpoolctl>=2.0.0 from https://files.pythonhosted.org/packages/81/12/fd4dea011af9d69e1cad05c75f3f7202cdcbeac9b712eea58ca779a72865/threadpoolctl-3.2.0-py3-none-any.whl.metadata
  Using cached threadpoolctl-3.2.0-py3-none-any.whl.metadata (10.0 kB)
Collecting pandas (from torchvf->omnipose)
  Obtaining dependency information for pandas from https://files.pythonhosted.org/packages/9e/71/756a1be6bee0209d8c0d8c5e3b9fc72c00373f384a4017095ec404aec3ad/pandas-2.0.3-cp311-cp311-win_amd64.whl.metadata
  Using cached pandas-2.0.3-cp311-cp311-win_amd64.whl.metadata (18 kB)
Collecting pyyaml (from torchvf->omnipose)
  Obtaining dependency information for pyyaml from https://files.pythonhosted.org/packages/b3/34/65bb4b2d7908044963ebf614fe0fdb080773fc7030d7e39c8d3eddcd4257/PyYAML-6.0.1-cp311-cp311-win_amd64.whl.metadata
  Using cached PyYAML-6.0.1-cp311-cp311-win_amd64.whl.metadata (2.1 kB)
Collecting torchvision (from torchvf->omnipose)
  Using cached torchvision-0.15.2-cp311-cp311-win_amd64.whl (1.2 MB)
Collecting six>=1.5 (from python-dateutil>=2.7->matplotlib->omnipose)
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Requirement already satisfied: MarkupSafe>=2.0 in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from jinja2->torch>=1.10->omnipose) (2.1.3)
Collecting pytz>=2020.1 (from pandas->torchvf->omnipose)
  Using cached pytz-2023.3-py2.py3-none-any.whl (502 kB)
Collecting tzdata>=2022.1 (from pandas->torchvf->omnipose)
  Using cached tzdata-2023.3-py2.py3-none-any.whl (341 kB)
Requirement already satisfied: mpmath>=0.19 in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from sympy->torch>=1.10->omnipose) (1.3.0)
Collecting pytorch-ranger>=0.1.1 (from torch-optimizer->cellpose-omni[omni]>=0.9.1->omnipose)
  Using cached pytorch_ranger-0.1.1-py3-none-any.whl (14 kB)
Collecting requests (from torchvision->torchvf->omnipose)
  Obtaining dependency information for requests from https://files.pythonhosted.org/packages/70/8e/0e2d847013cb52cd35b38c009bb167a1a26b2ce6cd6965bf26b47bc0bf44/requests-2.31.0-py3-none-any.whl.metadata
  Using cached requests-2.31.0-py3-none-any.whl.metadata (4.6 kB)
Collecting colorama (from tqdm->cellpose-omni[omni]>=0.9.1->omnipose)
  Using cached colorama-0.4.6-py2.py3-none-any.whl (25 kB)
Collecting charset-normalizer<4,>=2 (from requests->torchvision->torchvf->omnipose)
  Obtaining dependency information for charset-normalizer<4,>=2 from https://files.pythonhosted.org/packages/91/6e/db0e545302bf93b6dbbdc496dd192c7f8e8c3bb1584acba069256d8b51d4/charset_normalizer-3.2.0-cp311-cp311-win_amd64.whl.metadata
  Using cached charset_normalizer-3.2.0-cp311-cp311-win_amd64.whl.metadata (31 kB)
Collecting idna<4,>=2.5 (from requests->torchvision->torchvf->omnipose)
  Using cached idna-3.4-py3-none-any.whl (61 kB)
Collecting urllib3<3,>=1.21.1 (from requests->torchvision->torchvf->omnipose)
  Obtaining dependency information for urllib3<3,>=1.21.1 from https://files.pythonhosted.org/packages/9b/81/62fd61001fa4b9d0df6e31d47ff49cfa9de4af03adecf339c7bc30656b37/urllib3-2.0.4-py3-none-any.whl.metadata
  Using cached urllib3-2.0.4-py3-none-any.whl.metadata (6.6 kB)
Collecting certifi>=2017.4.17 (from requests->torchvision->torchvf->omnipose)
  Obtaining dependency information for certifi>=2017.4.17 from https://files.pythonhosted.org/packages/4c/dd/2234eab22353ffc7d94e8d13177aaa050113286e93e7b40eae01fbf7c3d9/certifi-2023.7.22-py3-none-any.whl.metadata
  Using cached certifi-2023.7.22-py3-none-any.whl.metadata (2.2 kB)
Using cached igraph-0.10.6-cp39-abi3-win_amd64.whl (2.9 MB)
Using cached matplotlib-3.7.2-cp311-cp311-win_amd64.whl (7.5 MB)
Using cached numba-0.57.1-cp311-cp311-win_amd64.whl (2.6 MB)
Using cached numpy-1.24.4-cp311-cp311-win_amd64.whl (14.8 MB)
Using cached scikit_image-0.21.0-cp311-cp311-win_amd64.whl (22.8 MB)
Using cached scipy-1.11.1-cp311-cp311-win_amd64.whl (44.0 MB)
Using cached scikit_learn-1.3.0-cp311-cp311-win_amd64.whl (9.2 MB)
Using cached torchvf-0.1.3-py3-none-any.whl (56 kB)
Using cached contourpy-1.1.0-cp311-cp311-win_amd64.whl (470 kB)
Using cached fonttools-4.41.1-cp311-cp311-win_amd64.whl (2.1 MB)
Using cached imageio-2.31.1-py3-none-any.whl (313 kB)
Using cached joblib-1.3.1-py3-none-any.whl (301 kB)
Using cached lazy_loader-0.3-py3-none-any.whl (9.1 kB)
Using cached llvmlite-0.40.1-cp311-cp311-win_amd64.whl (27.7 MB)
Using cached Pillow-10.0.0-cp311-cp311-win_amd64.whl (2.5 MB)
Using cached threadpoolctl-3.2.0-py3-none-any.whl (15 kB)
Using cached tifffile-2023.7.18-py3-none-any.whl (221 kB)
Using cached fastremap-1.13.5-cp311-cp311-win_amd64.whl (486 kB)
Using cached imagecodecs-2023.7.10-cp311-cp311-win_amd64.whl (24.3 MB)
Using cached natsort-8.4.0-py3-none-any.whl (38 kB)
Using cached opencv_python_headless-4.8.0.74-cp37-abi3-win_amd64.whl (38.0 MB)
Using cached pandas-2.0.3-cp311-cp311-win_amd64.whl (10.6 MB)
Using cached PyYAML-6.0.1-cp311-cp311-win_amd64.whl (144 kB)
Using cached requests-2.31.0-py3-none-any.whl (62 kB)
Using cached certifi-2023.7.22-py3-none-any.whl (158 kB)
Using cached charset_normalizer-3.2.0-cp311-cp311-win_amd64.whl (96 kB)
Using cached urllib3-2.0.4-py3-none-any.whl (123 kB)
Installing collected packages: texttable, pytz, urllib3, tzdata, threadpoolctl, six, pyyaml, pyparsing, pillow, pbr, packaging, numpy, natsort, llvmlite, lazy_loader, kiwisolver, joblib, igraph, idna, fonttools, cycler, colorama, charset-normalizer, certifi, tqdm, tifffile, scipy, requests, PyWavelets, python-dateutil, opencv-python-headless, numba, mgen, imageio, imagecodecs, fastremap, edt, contourpy, torchvision, scikit-learn, scikit-image, pytorch-ranger, peakdetect, pandas, matplotlib, cellpose-omni, torchvf, torch-optimizer, ncolor, omnipose
  Attempting uninstall: numpy
    Found existing installation: numpy 1.25.1
    Uninstalling numpy-1.25.1:
      Successfully uninstalled numpy-1.25.1
Successfully installed PyWavelets-1.4.1 cellpose-omni-0.9.1 certifi-2023.7.22 charset-normalizer-3.2.0 colorama-0.4.6 contourpy-1.1.0 cycler-0.11.0 edt-2.3.1 fastremap-1.13.5 fonttools-4.41.1 idna-3.4 igraph-0.10.6 imagecodecs-2023.7.10 imageio-2.31.1 joblib-1.3.1 kiwisolver-1.4.4 lazy_loader-0.3 llvmlite-0.40.1 matplotlib-3.7.2 mgen-1.2.1 natsort-8.4.0 ncolor-1.2.1 numba-0.57.1 numpy-1.24.4 omnipose-0.4.4 opencv-python-headless-4.8.0.74 packaging-23.1 pandas-2.0.3 pbr-5.11.1 peakdetect-1.2 pillow-10.0.0 pyparsing-3.0.9 python-dateutil-2.8.2 pytorch-ranger-0.1.1 pytz-2023.3 pyyaml-6.0.1 requests-2.31.0 scikit-image-0.21.0 scikit-learn-1.3.0 scipy-1.11.1 six-1.16.0 texttable-1.6.7 threadpoolctl-3.2.0 tifffile-2023.7.18 torch-optimizer-0.3.0 torchvf-0.1.3 torchvision-0.15.2 tqdm-4.65.0 tzdata-2023.3 urllib3-2.0.4
```

FInally, I tried to run wit with the GUI
```
python -m omnipose
GUI ERROR: No module named 'PyQt6'
GUI dependencies may not be installed (normal for first run).
Install GUI dependencies? (Note: uses PyQt6.) (y/n): y
```

I accepted, but it failed to install the omnipose GUI
```
Requirement already satisfied: cellpose-omni[gui] in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (0.9.1)
Requirement already satisfied: numpy>=1.22.4 in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from cellpose-omni[gui]) (1.24.4)
Requirement already satisfied: scipy in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from cellpose-omni[gui]) (1.11.1)
Requirement already satisfied: natsort in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from cellpose-omni[gui]) (8.4.0)
Requirement already satisfied: tifffile in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from cellpose-omni[gui]) (2023.7.18)
Requirement already satisfied: tqdm in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from cellpose-omni[gui]) (4.65.0)
Requirement already satisfied: numba in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from cellpose-omni[gui]) (0.57.1)
Requirement already satisfied: torch>=1.6 in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from cellpose-omni[gui]) (2.0.1)
Requirement already satisfied: opencv-python-headless in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from cellpose-omni[gui]) (4.8.0.74)
Requirement already satisfied: fastremap in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from cellpose-omni[gui]) (1.13.5)
Requirement already satisfied: imagecodecs in c:\gbw_myprograms\mambaforge\envs\omnipose\lib\site-packages (from cellpose-omni[gui]) (2023.7.10)
Collecting pyqtgraph>=0.12.4 (from cellpose-omni[gui])
  Using cached pyqtgraph-0.13.3-py3-none-any.whl (960 kB)
Collecting PyQt6.sip (from cellpose-omni[gui])
  Obtaining dependency information for PyQt6.sip from https://files.pythonhosted.org/packages/07/b6/94eb58d1c5f8467d4073b2bcd0d341e16a7082b00975d7fc77885db02878/PyQt6_sip-13.5.2-cp311-cp311-win_amd64.whl.metadata
  Downloading PyQt6_sip-13.5.2-cp311-cp311-win_amd64.whl.metadata (524 bytes)
Collecting PyQt6 (from cellpose-omni[gui])
  Obtaining dependency information for PyQt6 from https://files.pythonhosted.org/packages/0e/83/b5f0d005bb7d5bae034edadcb62dc70f9a1c242ed49eb333850136dc0b5c/PyQt6-6.5.2-cp37-abi3-win_amd64.whl.metadata
  Downloading PyQt6-6.5.2-cp37-abi3-win_amd64.whl.metadata (2.2 kB)
Collecting google-cloud-storage (from cellpose-omni[gui])
  Obtaining dependency information for google-cloud-storage from https://files.pythonhosted.org/packages/88/14/c9d4faae7ea4bff4405152cbc762b61100aa6949273b4eb3203d23308670/google_cloud_storage-2.10.0-py2.py3-none-any.whl.metadata
  Downloading google_cloud_storage-2.10.0-py2.py3-none-any.whl.metadata (6.0 kB)
INFO: pip is looking at multiple versions of cellpose-omni[gui] to determine which version is compatible with other requirements. This could take a while.
Collecting cellpose-omni[gui]
  Using cached cellpose_omni-0.7.3-py3-none-any.whl (196 kB)
  Using cached cellpose_omni-0.7.2-py3-none-any.whl (196 kB)
  Using cached cellpose_omni-0.7.0-py3-none-any.whl (195 kB)
  Using cached cellpose_omni-0.6.9-py3-none-any.whl (195 kB)
Collecting opencv-python==4.5.5.64 (from cellpose-omni[gui])
  Using cached opencv_python-4.5.5.64-cp36-abi3-win_amd64.whl (35.4 MB)
Collecting cellpose-omni[gui]
  Using cached cellpose_omni-0.6.8-py3-none-any.whl (195 kB)
  Using cached cellpose_omni-0.6.7-py3-none-any.whl (195 kB)
ERROR: Cannot install cellpose-omni[gui]==0.6.7, cellpose-omni[gui]==0.6.8, cellpose-omni[gui]==0.6.9, cellpose-omni[gui]==0.7.0, cellpose-omni[gui]==0.7.2, cellpose-omni[gui]==0.7.3 and cellpose-omni[gui]==0.9.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    cellpose-omni[gui] 0.9.1 depends on omnipose-theme; extra == "gui"
    cellpose-omni[gui] 0.7.3 depends on omnipose-theme; extra == "gui"
    cellpose-omni[gui] 0.7.2 depends on omnipose-theme; extra == "gui"
    cellpose-omni[gui] 0.7.0 depends on omnipose-theme; extra == "gui"
    cellpose-omni[gui] 0.6.9 depends on omnipose-theme; extra == "gui"
    cellpose-omni[gui] 0.6.8 depends on omnipose-theme; extra == "gui"
    cellpose-omni[gui] 0.6.7 depends on omnipose-theme; extra == "gui"

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\GBW_MyPrograms\mambaforge\envs\omnipose\Lib\site-packages\omnipose\__main__.py", line 3, in <module>
    main(omni_CLI=True)
  File "C:\GBW_MyPrograms\mambaforge\envs\omnipose\Lib\site-packages\cellpose_omni\__main__.py", line 191, in main
    install('cellpose-omni[gui]')
  File "C:\GBW_MyPrograms\mambaforge\envs\omnipose\Lib\site-packages\cellpose_omni\__main__.py", line 34, in install
    subprocess.check_call([sys.executable, "-m", "pip", "install", package])
  File "C:\GBW_MyPrograms\mambaforge\envs\omnipose\Lib\subprocess.py", line 413, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['C:\\GBW_MyPrograms\\mambaforge\\envs\\omnipose\\python.exe', '-m', 'pip', 'install', 'cellpose-omni[gui]']' returned non-zero exit status 1.
```

Any idea?

**Comment by kevinjohncutler** [comment]
I have not seen this error before, but maybe (1) could be mamba vs conda or (2) could be a python version issue. I think (2) is more likely. Try making a new environment with a specific python version, one that is NOT the same as your base python. This will make sure it is not trying to use packages outside of your environment (`which python` and `python -V` should confirm this).

**Comment by ebornet** [comment]
Hi, 
I had a similar error on Linux
https://github.com/kevinjohncutler/omnipose/issues/50 - this is how I fixed the error at least on Linux

**Comment by kevinjohncutler** [comment]
@ebornet thanks for the solution for #50. Turns out I had to use that exact same command for a GUI issue with different output messages last year, only on Ubuntu 22.04 and later. For reference to @bpavie and others, the command is:

`sudo apt-get install '^libxcb.*-dev' libx11-xcb-dev libglu1-mesa-dev libxrender-dev libxi-dev libxkbcommon-dev libxkbcommon-x11-dev `

I am not sure what the analog is on windows. However, I will also recommend trying a new environment set up with the most recent torch/cuda and more recent python:  

`conda create -n omnipose 'python==3.10' pytorch torchvision pytorch-cuda=11.8 -c pytorch -c nvidia`
